src/pybamm/models/full_battery_models/lithium_ion/electrode_soh_half_cell.py

```diff
@@ -49,13 +49,12 @@ def __init__(self, name="ElectrodeSOH model"):
             "Uw(x_100)": U_w(x_100, T_ref),
             "Uw(x_0)": U_w(x_0, T_ref),
             "Q_w": Q_w,
-            "Q_w * (x_100 - x_0)": Q_w * (x_100 - x_0),
         }
 
     @property
     def default_solver(self):
         # Use AlgebraicSolver as CasadiAlgebraicSolver gives unnecessary warnings
-        return pybamm.AlgebraicSolver()
+        return pybamm.AlgebraicSolver(method="minimize  L-BFGS-B", tol=1e-7)
```

Comment:
I tried this out on the example linked in your issue. Using the default tol (1e-6) seems to work reliably for me, is there a reason to tighten this?

Reply:
I changed the tolerance because a regression test was failing.